scripts/cv_api_master.py

```diff
@@ -0,0 +1,155 @@
+PREPROCESSING_MASTER = {
+    'path': 'preprocessing/',
+    'title': 'Preprocessing Layers',
+    'toc': True,
+    'children': [
+        {
+            'path': 'auto_contrast',
+            'title': 'The AutoContrast class',
```

Comment:
- Remove the "The" prefixes
- Mention "layer" since they're layers
- Note that the "class" suffix gets added automatically when pulling the object to documented, e.g. see this page https://keras.io/api/layers/core_layers/dense/
- So here just "AutoContrast layer"

Reply:
Ah thank you, updated.